admin-dev/themes/new-theme/js/product-page/nested-categories.js

```diff
@@ -1,6 +1,8 @@
 /**
  * Nested categories management
  */
+import defaultCategory from '../../../default/js/bundle/product/default-category';
```

Comment:
doing this, we create a (too)strong dependency between the 2 themes
WDYT @NeOMakinG ?

Reply:
Totally agree with you @sowbiba . You confirmed to me that this PR should be closed.
Thanks a lot for your assistance.